logging/google/cloud/logging/handlers/middleware/__init__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
If this is a Django middleware, then it is customary to have `middleware` in the package name as she has done.

Also, our entire approach to the package source hierarchy is overboard. This point is, at best, the pot calling the kettle black.